client/hbs/editor.hbs

```diff
@@ -34,9 +34,20 @@
 				</button>
 			</footer>
 		</main>
+		<div class='uploader'>
```

Comment:
afaict `.upload-item` is a useless selector.  Everything inside `.uploader` (which we might want to be `#uploader`?) is a `.upload-item`, and nothing else is.